enhancements/cluster-logging/forward-to-loki.md

```diff
@@ -148,6 +148,12 @@ The following optional output fields are Loki-specific:
   Example: `kubernetes.labels.foo` => `kubernetes_labels_foo`.\
   **Note**: `kubernetes.host` is *always* be included, even if not requested.
   It is required to ensure ordered label streams.
+- `tenantKey`: (string, default=`"kubernetes.namespaceName"`) \
```

Comment:
We don't provide a default for in-cluster.

Reply:
In addition do we really want to define as tenant key a label key or simply leave it as arbitrary data?

Reply:
A fixed tenant ID limits us to static tenants pre-defined by the log admin based on input filters. This has turned out to be inadequate for other cases (JSON, cloudwatch groups etc.), where the logging system needs to respond to new label values or new namespaces appearing in the cluster without a manual re-config each time. I think the same applies to tenants - you can't implement "namespace as tenant" without it unless you know all your namespaces in advance.